src/Configuration/Programmer/Programmer_cmd.cpp

```diff
@@ -19,57 +19,145 @@ You should have received a copy of the GNU General Public License
 along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
+#include <iostream>
+
 #include "CFGCommon/CFGCommon.h"
 #include "Programmer.h"
 
+void progressCallback(std::string msg) { std::cout << msg; }
+
+void messageCallback(std::string msg) { std::cout << "MS-CALLBACK :" << msg; }
```

Comment:
Same: should print const char* not std::string.

And every message that printed will have "MS-CALLBACK :" in front?!?! Do you think it is good idea?

Reply:
I will change the std::cout to CFG_POST_MSG. The prefix is just for me to identify the message. It is not necessary.